src/themes/retro-swirly.css

```diff
@@ -0,0 +1,167 @@
+	--col: var(--sl-col-width, 10px);
+	--width: calc(7 * var(--col));
+	--ring1: calc(var(--col) * 1);
+	--ring2: calc(var(--col) * 2);
+	--ring3: calc(var(--col) * 3);
+	--ring4: calc(var(--col) * 4);
+	--ring5: calc(var(--col) * 5);
+	--ring6: calc(var(--col) * 6);
+	--ring7: calc(var(--col) * 7);
+	--col1: calc(var(--offset) + var(--ring1));
+	--col2: calc(var(--offset) + var(--ring2));
+	--col3: calc(var(--offset) + var(--ring3));
+	--col4: calc(var(--offset) + var(--ring4));
+	--col5: calc(var(--offset) + var(--ring5));
+	--col6: calc(var(--offset) + var(--ring6));
+	--col7: calc(var(--offset) + var(--ring7));
+	--offset-width: calc(var(--offset) + var(--width));
+	--offset-width-rings: calc(var(--offset-width) + (4 * var(--col)));
+  
+	--i-linear-gradient: linear-gradient(
+	  to right,
+	  transparent 0 var(--offset),
+	  var(--sl-color1, yellow) var(--offset) var(--col1),
+	  var(--sl-color2, orange) var(--col1) var(--col2),
+	  var(--sl-color3, darkorange) var(--col2) var(--col3),
+	  var(--sl-color4, darkred) var(--col3) var(--col4),
+	  var(--sl-color3, darkorange) var(--col4) var(--col5),
+	  var(--sl-color2, orange) var(--col5) var(--col6),
+	  var(--sl-color1, yellow) var(--col6) var(--col7),
+	  transparent var(--col7)
+	);
+  
+	--right: var(--sl-left) calc(var(--offset) - 2px);
+  
+	--i-radial-gradient: radial-gradient(
+	  circle at var(--right, var(--offset-width)) var(--v-offset),
+	  var(--sl-color1, yellow) 0 var(--ring1),
+	  var(--sl-color2, orange) var(--ring1) var(--ring2),
+	  var(--sl-color3, darkorange) var(--ring2) var(--ring3),
+	  var(--sl-color4, darkred) var(--ring3) var(--ring4),
+	  transparent var(--ring4)
+	);
+  
+	background: var(--i-linear-gradient) 0 var(--v-offset) /
+		var(--offset-width-rings) 100%,
+	  var(--i-radial-gradient) top 0 left 0 / var(--offset-width-rings) 100%,
+	  var(--i-linear-gradient) top 0 left 0 / var(--offset-width-rings) 100%;
+	background-repeat: no-repeat;
+	background-attachment: local;
+	background-origin: border-box;
+	position:fixed;
+	top:0;
+	left:0;
+	content:'';
+	min-height: 100vh;
+	min-width:100vw;
+	pointer-events: none;
+}
+
+[data-twitch-message] {
+	line-height: 22.5px;
+	margin: 0;
+	padding-bottom: 5.625px;
+	padding-left: 22.5px;
+	padding-right: 22.5px;
+	padding-top: 5.625px;
+}
+
+[data-twitch-message-reply] {
+	color: var(--twitch-gray);
+	display: block;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+    overflow: hidden;
+}
+
+[data-twitch-message] .twitch-chat-sender {
+	color: var(--twitch-sender-color, var(--twitch-purple));
+	display: inline;
+	font-weight: 700;
+	font-family: "MuseoModerno", cursive;
+}
+
+[data-twitch-message] .twitch-chat-sender:after {
+	content: ": ";
+	color: var(--twitch-dark-background);
+	font-weight: normal;
+}
+
+[data-twitch-message] .twitch-chat-message {
+	display: inline;
+}
+
+[data-twitch-emote] {
+	width: 1.2em;
+}
+
+[data-twitch-mentioned-user] {
+	background-color: #dedee3;
```

Comment:
Just checking - do we want to leave the mentions with this gray background? It's totally cool if yes, just want to make sure.

Reply:
I actually redid the styles a little bit on this one. the contrast on the user text against a light blue background wasn't great. So I updated it to use cards.

Reply:
Ahh, I like the card treatment! 👍🏻